tests/test_utils/test_tx_utils.py

```diff
@@ -111,3 +113,49 @@ def test_get_latest_base_fee(fee):
     if type(fee) == str and fee.startswith("0x"):
         fee = int(fee, 0)
     assert get_latest_base_fee(web3) == int(fee)
+
+
+def test_get_effective_gas_price__eth():
```

Comment:
Do we want to add unhappy for eth and arb?  Looks like only set up for polygon currently

Reply:
Yep, so unhappy path there was added just because polygon is using external API to calc gas

Reply:
For other chains we don't call http client here

Reply:
Makes sense 👍